backup-loop.sh

```diff
@@ -30,6 +30,9 @@ fi
 : "${XDG_CONFIG_HOME:=/config}" # for rclone's base config path
 : "${ONE_SHOT:=false}"
 : "${TZ:=Etc/UTC}"
+: "${RCLONE_REPOSITORY:={DEST_DIR}"
```

Comment:
Should this be
```suggestion
: "${RCLONE_REPOSITORY:=${DEST_DIR}}"
```

...however it also looks like that variable is not used or documented.

Reply:
Sorry, not very fluent in bash yet. I'm trying to make the `RCLONE_REPOSITORY` default to /backups, or if the user used the `DEST_DIR` variable instead of `RCLONE_REPOSITORY`, then it would still work as they expected.